soh/soh/GameMenuBar.cpp

```diff
@@ -722,7 +722,7 @@ namespace GameMenuBar {
                     UIWidgets::EnhancementRadioButton("Rotate Link with Right Stick", "gPauseLiveLinkRotation", 3);
                     UIWidgets::Tooltip("Allow you to rotate Link on the Equipment menu with the Right Stick\nYou can zoom in by pointing up and reset Link's rotation by pointing down");
                     if (CVarGetInteger("gPauseLiveLinkRotation", 0) != 0) {
-                        UIWidgets::EnhancementSliderInt("Rotation Speed: %d", "##MinRotationSpeed", "gPauseLiveLinkRotationSpeed", 1, 20, "");
+                        UIWidgets::EnhancementSliderInt("Rotation Speed: %d", "##MinRotationSpeed", "gPauseLiveLinkRotationSpeed", 1, 20, "", 1);
```

Comment:
not clear what adding this 1 is doing here

Reply:
That's a default value.  For some reason they never put a default value in, which is bad practice.